src/app/features/activity-list/activity-list.tsx

```diff
@@ -2,7 +2,7 @@ import { useMemo } from 'react';
 
 import { useGetAccountTransactionsWithTransfersQuery } from '@app/query/stacks/transactions/transactions-with-transfers.query';
 import { useGetBitcoinTransactionsByAddressQuery } from '@app/query/bitcoin/address/transactions-by-address.query';
-import { useStacksPendingTransactions } from '@app/query/stacks/mempool/mempool.hooks';
+import { useStacksPendingTransactionsQuery } from '@app/query/stacks/mempool/mempool.hooks';
```

Comment:
Typically, I was using `Query` in the naming only if it was the hook exposing the actual query.

Reply:
So, in this case, I would have changed `useAccountMempool` to `useAccountMempoolQuery` instead.

Reply:
For the record, I didn't see the entire query is being returned here from the hook so can ignore this.

Reply:
If it helps can drop the `*Query` or rename to `*TransformedQuery`

Reply:
+1 to Fara's comment. Would keep with convention that only hooks that contain a `useQuery` use query naming